spec/features/further_education_payments/providers/provider_verifying_claims_spec.rb

```diff
@@ -63,6 +63,21 @@
 
       click_on "Save and continue"
 
+      # Performance and discipline
```

Comment:
not an issue but i find checking the `h1` makes for debugging issues quicker in future where you may be on a wrong page for whatever reason